CITATION.cff

```diff
@@ -0,0 +1,65 @@
+cff-version: 1.2.0
```

Comment:
This cff versioning is new to me. Why is it 1.2.0?

Reply:
To my knowledge it's the latest version of the [standard](https://citation-file-format.github.io/)